examples/routing_service/remote_admin/c++11/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 #  use or inability to use the software.
 #
 cmake_minimum_required(VERSION 3.11)
-project (RoutingServiceRemoteAdmin)
+project (RoutingServiceRemoteAdminC++11)
```

Comment:
Why did you change the project name? We use the same name in all the languages for a single example.

Reply:
But there will be no name collisions since we will not import one example into another.

Reply:
In any case, the examples related to the services are very inconsistent and may need a big rewrite. For now, we can leave them like that and in the future redesign all of them.